test/python/test_bsc.py

```diff
@@ -41,9 +41,17 @@ def perform_final_tx(self, destination, send_amount, fees, memo):
             pass
         # TODO : assert signature validity
 
+
 # Use a class to reuse the same Speculos instance
 class TestsBSC:
+    @pytest.mark.parametrize('test_to_run', ALL_TESTS_EXCEPT_MEMO)
+    def test_bsc(self, backend, exchange_navigation_helper, test_to_run):
+        BSCTests(backend, exchange_navigation_helper).run_test(test_to_run)
+
 
+class TestsBSC_EVM:
     @pytest.mark.parametrize('test_to_run', ALL_TESTS_EXCEPT_MEMO)
     def test_bsc(self, backend, exchange_navigation_helper, test_to_run):
+        # Override CAL to use the Ethereum app for BSC instead of its clone
+        TICKER_ID_TO_CONF["BNB"] = BSC_CONF_EVM
```

Comment:
Not a huge fan, overriding globals like this is hard to track, but I don't see an easier way to do it so...

Maybe I'll rework the `ExchangeTestRunner` so that it takes the conf as input instead of deducing it, this would solve the issue.. Problem for later